.github/prompts/add-unit-tests.prompt.md

```diff
@@ -0,0 +1,505 @@
+Add unit tests for this Go file in the Terraform AzureRM provider. Make sure you cover the important use cases including resource lifecycle operations, Azure API integration, and edge cases. The code must build and pass tests without errors.
```

Comment:
Do we need to generate prompt files for resource/data source, resource/data source document, and acctest respectively?

Reply:
No, you can just tell the chat agent to "add acceptance tests for this thing1" and it will do it for you.

Reply:
Why can’t we use generate prompt files to create resources/data sources, resource/data source documentation, and acceptance tests? Wouldn’t that be more convenient for users?